.github/actions/release-version/action.yml

```diff
@@ -0,0 +1,142 @@
+name: 'Release version'
```

Comment:
This action is not yet being consumed. What it does it look at a release branch name, like release/0.1 and provides back the next release or prerelease version.

For example a branch like release/0.1, if there is already 0.1.0 and 0.1.1-rc.0 published should return the next version as 0.1.1 and the next prerelease as 0.1.1-rc.1